### ISSUE TITLE
no-unused-variable with import used in other imports

### ISSUE THREAD

**Issue by praveennet** [palantir/tslint#651]
I'm getting false positive when using this rule 'no-unused-variable' for the below code. Could this be fixed please?

```typescript
import ExternalDefinition = require("ExternalDefinition");
import Def = ExternalDefinition.MyNamespace.MyClass1;

export class MyImplementation
    extends MyBaseClasses.BaseClass1
    implements Def.Contract {

}
```

Error: 
> TsLint: unused variable: 'ExternalDefinition'

**Comment by JKillian** [comment]
Hi @praveennet, thanks for the report. I think your issue is the same as #570. Feel free to comment on that one with your code sample!

**Comment by praveennet** [comment]
But here import is declared in same level and used in next line itself. it looks different from #570 to me. ?

**Comment by adidahiya** [comment]
@praveennet your issue is actually a duplicate of #613. I'm not sure if @JKillian's PR (#655) fixes this issue as well; we'll need to investigate.

**Comment by JKillian** [comment]
Interestingly, I found that #655 fixes both cases of #613 but not this issue. Going to reopen this

**Comment by JKillian** [comment]
Actually, this is a duplicate of #325 :smiley:
